packages/react/src/lib/useVoiceClient.ts

```diff
@@ -134,23 +122,26 @@ export const useVoiceClient = (props: {
                 content: string;
               }) => ({
                 type: 'tool_error',
-                tool_call_id: message.tool_call_id,
+                toolCallId: message.toolCallId,
                 error,
                 code,
-                level,
+                level: level != null ? 'warn' : undefined, // level can only be warn
                 content,
               }),
             })
             .then((response) => {
               // check that response is a correctly formatted response or error payload
-              const parsed = z
-                .union([ToolResponseSchema, ToolErrorSchema])
-                .safeParse(response);
+              const parsedResponse =
+                serializers.empathicVoice.ToolResponseMessage.parse(response);
+              const parsedError =
+                serializers.empathicVoice.ToolErrorMessage.parse(response);
 
               // if valid send it to the socket
               // otherwise, report error
-              if (parsed.success) {
-                client.current?.sendToolMessage(parsed.data);
+              if (response.type === 'tool_response' && parsedResponse.ok) {
```

Comment:
meeting note: Deep tried to make this as back-compat as possible by keeping our custom logic for merging tool call messages. In the future we could consider making the React SDK simpler by treating these as two unrelated message types. For now we will retain the convenience functionality